go.mod

```diff
@@ -4,4 +4,6 @@ require (
 	github.com/google/go-cmp v0.2.0 // test dependency
 	github.com/influxdata/flux v0.0.0-20190620184636-886e3c28388d // test dependency
 	github.com/influxdata/line-protocol v0.0.0-20190509173118-5712a8124a9a
+	github.com/stretchr/testify v1.3.0
```

Comment:
add  // test dependency 
comment.  It helps when keeping track.

Reply:
nice! will do that!